flashinfer_bench/serve/low_level/executor.py

```
@@ -0,0 +1,658 @@
+
+        Returns
+        -------
+        None
+            This method releases request-owned resources in place.
+        """
+        self.register_file.clear()
+        for function_name in sorted(self.registered_function_names):
+            try:
+                tvm_ffi.remove_global_func(function_name)
+            except Exception:
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)